src/ansys/units/quantity.py

```diff
@@ -137,10 +137,12 @@ def __init__(
         if not isinstance(units, Unit):
             units = Unit(units)
 
+        min_value = value if isinstance(value, float) else min(value)
```

Comment:
Correct. 👍 

Thanks a lot.

Please could you add a test for this in https://github.com/ansys/pyansys-units/blob/main/tests/test_quantity.py so that we can track and avoid this issue in future.